SemanticSearchV2.md

```diff
@@ -264,30 +279,49 @@ The document response is divided into two parts. First is the parsed_input node
 				"weight" : 100
 				"relationships" : {},
 			},
-			{
-				"type" : "job_level",
-				"name" : "EXPERIENCED_NON_MANAGER"
-			},
-			{
-				"type" : "location",
-				"entity_node" : {
-					"address" : "Atlanta, Georgia",
-					"region" : "Georgia"
-				}
-			},
-			{
-				"name" : "Java developer",
-				"type" : "job_title"
-			}
+			...
 		],
-		"versions": {
-			"extracted_keywords": "ExtractedKeywords",
-			"job_titles": "CaroteneV3",
-			"skills": "SkillsV4",
-			"related_keywords": "RelatedSearchTermsV1",
-			"text_kernel_related_keywords": "TextKernelRelatedSearchTerms",
-			"custom_keywords": "CustomKeywords"
-		}
+		"location":{  
+		   "address":"Pleasanton, California, Norcross, Georgia",
```

Comment:
The spacing between colons is inconsistent. Sometimes you do "foo":"bar", sometimes "foo": "bar" and sometimes "foo" : "bar".